src/css/doc.css

```diff
@@ -509,19 +477,6 @@ ul ul ul {
   font-weight: inherit;
 }
 
-/* Use this for a working pseudo element before an [abstract] attribute-marked block in the documentation.
-  This one adds the words "Quick Summary" before any text inside the [abstract] block. */
-
-/*
-.doc .abstract blockquote::before {
-  content: "Quick Summary \a";
-  white-space: pre;
-  color: var(--color-muted);
-  font-weight: var(--weight-medium);
-  font-size: var(--heading-h4);
-  padding-bottom: 1rem;
-}  */
-
```

Comment:
You asked me to leave this in, in case we wanted to use it as a working example of a Pseudo :before in the future :) Sure you want to delete it?

Reply:
Hahaha, good catch.
(Trying to purge commented CSS as it adds to cognitive weight, but will have a think...)